pages/instructorIssues/instructorIssues.js

```diff
@@ -143,7 +143,7 @@ router.get('/', function(req, res, next) {
 });
 
 router.post('/', function(req, res, next) {
-    if (!res.locals.authz_data.has_instructor_edit) return next();
+    if (!res.locals.authz_data.has_course_permission_edit) return next();
```

Comment:
Won't this fail for users who are instructors but not course editors?

Reply:
Yes.

I could go either way on this (I had an "or" with `has_instructor_edit` for a while, then got rid of it). On the one hand, these issues are a course-level thing, resolving them requires editing course content and syncing these edits, etc. On the other hand, current practice might be to ask non-editors to resolve issues by pushing/pulling content to the course git repo. Thoughts?

Reply:
I don't have any strong thoughts about the "right" answer here.

But just in practice at the moment, I believe instructors-who-aren't-course-editors can see issues, right? And they used to be able to close issues? If so, I'm generally in favor of letting them continue to do this until the new perm system is merged, to avoid confusion. Once the new perm system is merged we can do a big news item explaining how it works, but until then I think it would be best to minimize the removal of capabilities.

Reply:
Sure, done